### PR TITLE
Updating model classes for Custom Analyzers API

### PR DESCRIPTION
**Pull request by Yahnoosh**

### CODE REVIEW

**Comment by Yahnoosh** [comment]
@brjohnstmsft

**Comment by Yahnoosh** [comment]
@brjohnstmsft is added to the review. #Closed

File: src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/NGramTokenFilter.cs [hunk 1]
```diff
@@ -40,7 +41,8 @@ public NGramTokenFilter(string name, int? minGram = default(int?), int? maxGram
         }
 
         /// <summary>
-        /// Gets or sets the minimum n-gram length. Default is 1.
+        /// Gets or sets the minimum n-gram length. Default is 1. Must be less
+        /// than minGram.
```
Comment (Yahnoosh) [inline src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/NGramTokenFilter.cs]: > minGram [](start = 17, length = 7)

this is wrong #Resolved

File: src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/EdgeNGramTokenFilter.cs [hunk 2]
```diff
@@ -42,7 +43,8 @@ public EdgeNGramTokenFilter(string name, int? minGram = default(int?), int? maxG
         }
 
         /// <summary>
-        /// Gets or sets the minimum n-gram length. Default is 1.
+        /// Gets or sets the minimum n-gram length. Default is 1. Must be less
+        /// than the value of maxGram.
```
Comment (Yahnoosh) [inline src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/EdgeNGramTokenFilter.cs:47]: > maxGram [](start = 30, length = 7)

make uppercase

File: src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/ShingleTokenFilter.cs [hunk 3]
```diff
@@ -50,7 +50,8 @@ public ShingleTokenFilter(string name, int? maxShingleSize = default(int?), int?
         public int? MaxShingleSize { get; set; }
 
         /// <summary>
-        /// Gets or sets the minimum shingle size. Default is 2.
+        /// Gets or sets the minimum shingle size. Default is 2.  Must be less
+        /// than the value of maxShingleSize.
```
Comment (Yahnoosh) [inline src/Search/Microsoft.Azure.Search/GeneratedSearchService/Models/ShingleTokenFilter.cs:54]: > maxShingleSize [](start = 30, length = 14)

uppercase

**Comment by bitsforthought** [comment]
Please re-create this PR such that it targets Azure/search-v2-preview-dev instead of Yahnoosh/search-v2-preview-dev.

**Pull request closed without merging**